tests/functional/includes/fn.bind.polyfill.js

```diff
@@ -0,0 +1,51 @@
+// Needed for PhantomJS < 2.0
+
+if (!Function.prototype.bind) {
```

Comment:
Is there an npm package we can add to `devDependencies` for this? /cc @juandopazo